src/models/PlayerStats.ts

```diff
@@ -1,10 +1,13 @@
 import ActiveModel from '#models/ActiveModel.js';
 import serializer from '#utils/serializer.js';
 
+const DEFAULT_RATING = 750.0;
+
 export default class PlayerStats extends ActiveModel {
   protected data: {
     playerId: string
     stats: Map<string, any>
+    ratings: Map<string, any>
```

Comment:
While doing it your way is acceptable, I actually intended to place the ratings in the "stats" structure.  It's admittedly confusing.  The "stats" map keys are player IDs.  If the stats are vs someone else, then the key is their player ID.  If you want your own stats, it is stored under YOUR player ID (and matches the playerId property).  I currently track the "completed" games and "abandoned" games in this location.

Reply:
Ah I see. I can do it that way, keying it off their self